cluster/cluster.go

```diff
@@ -51,7 +51,7 @@ type Cluster interface {
 
 	// Return some info about the cluster, like nb or containers / images
 	// It is pretty open, so the implementation decides what to return.
-	Info() [][2]string
+	Info() ([][]string, int64, int64)
```

Comment:
Nit (feel totally free to ignore it):

It might be cleaner to have `cluster.Memory()`

WDYT?